docs/README.md

```diff
@@ -39,30 +39,33 @@ Optionally to enable Login to the dashboard
 
 ### Create your GitHub App
 
-Create a GitHub app from your GitHub profile page.
+- Create a [GitHub app](https://developer.github.com/apps/building-your-first-github-app/#register-a-new-app-with-github) from your GitHub profile page. Enter the following required fields:
+  - GitHub App name
+  - Homepage URL
+  - Webhook URL
 
-Select these OAuth permissions: 
+    The GitHub app will deliver webhooks to your OpenFaaS Cloud instance every time code is pushed in a user's function repository. Make sure you provide the public URL for your OpenFaaS gateway to the GitHub app:
```

Comment:
Hey @codeboten everything is perfect  now in terms of content 💯 , in alignment, can we please align that text `The GitHub app ...` with `- Webhook URL` because it looks misaligned in terms of the text above:
![image](https://user-images.githubusercontent.com/34942004/49209937-85479280-f3c4-11e8-8679-ebc6663ba658.png)

I think the text will look better if it is part of the `Webhook URL` bullet and detail as sub bullets(as they are done now), what do you think?

Reply:
Updated. Thanks again for the feedback!